### PR TITLE
MOTECH-2028: Made unique editable from schema editor UI

### PR DESCRIPTION
**Pull request by pgesek**
* Only non DDE fields
* Only non collection types

### CODE REVIEW

**Comment by pgesek** [comment]
test this psql

File: platform/mds/mds/src/main/java/org/motechproject/mds/builder/impl/MDSConstructorImpl.java [hunk 1]
```diff
@@ -310,6 +313,32 @@ public void updateFields(Entity entity, Map<String, String> fieldNameChanges) {
         }
     }
 
+    @Override
+    @Transactional
```
Comment (tstalka) [inline platform/mds/mds/src/main/java/org/motechproject/mds/builder/impl/MDSConstructorImpl.java]: @Transactional("dataTransactionManager")

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by tstalka** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Pull request merged**